src/components/chat-view/Chat.tsx

```diff
@@ -118,9 +118,10 @@ const Chat = forwardRef<ChatRef, ChatProps>((props, ref) => {
   const [queryProgress, setQueryProgress] = useState<QueryProgressState>({
     type: 'idle',
   })
+  const [isStreaming, setIsStreaming] = useState(false)
```

Comment:
isn't it enough to use `submitMutation.isPending`?

Reply:
You're right. I'd fix it.